src/Animals.java

```diff
@@ -0,0 +1,19 @@
+public class Animals {
+    protected String name;
+    protected int obstacleLength;
```

Comment:
animals is a class that represent an entity, not entities, obstacleLength should not be here (this attribute is not relative to animals )